tests/test_flows/test_flow_functions.py

```diff
@@ -256,3 +256,23 @@ def test_sklearn_to_flow_list_of_lists(self):
         server_flow = openml.flows.get_flow(flow.flow_id, reinstantiate=True)
         self.assertEqual(server_flow.parameters['categories'], '[[0, 1], [0, 1]]')
         self.assertEqual(server_flow.model.categories, flow.model.categories)
+
+    def test_get_flow_reinstantiate_model(self):
```

Comment:
This appears to be a simple version of `tests.test_flows.test_flows:TestFlow.test_sklearn_to_upload_to_flow`. Could you please check that?

Reply:
But it might be good to have such a test next to the one below.

Reply:
I don't think `tests.test_flows.test_flows:TestFlow.test_sklearn_to_upload_to_flow` ever actually tests if the model is reinstantiated properly. I think the model is ignored by both `assert_flows_equal` and `_to_xml`.
Of course, it is easy to add the check there. But I think it's good to test for this separately from such a complicated flow setup. If this unit test fails, it's very likely due to an error in reinstantiation. If the other test fails, it can be due to any number of reasons, such as serializing subflows, generating xml, etc.